### PR TITLE
1.13.0-rc2 changelog

### PR DESCRIPTION
**Pull request by techknowlogick**
as title

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,35 @@ This changelog goes through all the changes that have been made in each release
...
+  * Ensure topics added using the API are added to the repository (#13285) (#13302)
+  * Fix avatar autogeneration (#13282)
+  * Add migrated pulls to pull request task queue (#13331) (#13334)
+  * Fix scrolling to resolved comment anchors (#13343) (#13371)
+  * When creating line diffs do not split within an html entity (#13357) (#13375) (#13425) (#13427)
+  * Fix reactions on code comments (#13390) (#13401) 
+  * Add missing full names when DEFAULT_SHOW_FULL_NAME is enabled (#13424)
+  * Replies to outdated code comments should also be outdated (#13217) (#13433)
+  * Fix panic bug in handling multiple references in commit (#13486) (#13487)
+  * Prevent panic on git blame by limiting lines to 4096 bytes at most (#13491)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Prevent panic on git blame by limiting lines to 4096 bytes at most (#13470) (#13491)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,35 @@ This changelog goes through all the changes that have been made in each release
...
-## [1.13.0-RC1](https://github.com/go-gitea/gitea/releases/tag/v1.13.0-rc1) - 2020-10-14
+## [1.13.0-rc2](https://github.com/go-gitea/gitea/releases/tag/v1.13.0-rc2) - 2020-11-10
+* ENHANCEMENTS
+  * Return the full rejection message and errors in flash errors (#13221) (#13237)
+  * Remove PAM from auth dropdown when unavailable (#13276) (#13281)
+* BUGFIXES
+  * Fix Italian language file parsing error (#13156)
+  * Show outdated comments in pull request (#13148) (#13162)
+  * Fix parsing of pre-release git version (#13169) (#13172)
+  * Fix diff skipping lines (#13155)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Fix diff skipping lines (#13154) (#13155)
```

**Review by jolheiser — APPROVED** [review]

**Review by 6543 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,35 @@ This changelog goes through all the changes that have been made in each release
...
+  * When handling errors in storageHandler check underlying error (#13178) (#13193)
+  * Fix size and clickable area on file table back link (#13205) (#13207)
+  * Add better error checking for inline html diff code (#13251)
+  * Fix initial commit page & binary munching problem (#13249) (#13258)
+  * Fix migrations from remote Gitea instances when configuration not set (#13229) (#13273)
+  * Store task errors following migrations and display them (#13246) (#13287)
+  * Fix bug isEnd detection on getIssues/getPullRequests (#13299) (#13301)
+  * When the git ref is unable to be found return broken pr (#13218) (#13303)
+  * Ensure topics added using the API are added to the repository (#13285) (#13302)
+  * Fix avatar autogeneration (#13282)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Fix avatar autogeneration (#13233) (#13282)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,7 +4,35 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add migrated pulls to pull request task queue (#13331) (#13334)
```
Comment (6543) [inline CHANGELOG.md:26]:
```suggestion
  * Add migrated pulls to pull request task queue (#13331) (#13334)
  * Remove obsolete change of email on profile page (#13341) (#13347)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,7 +4,36 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix size and clickable area on file table back link (#13205) (#13207)
+  * Add better error checking for inline html diff code (#13251)
+  * Fix initial commit page & binary munching problem (#13249) (#13258)
+  * Fix migrations from remote Gitea instances when configuration not set (#13229) (#13273)
+  * Store task errors following migrations and display them (#13246) (#13287)
+  * Fix bug isEnd detection on getIssues/getPullRequests (#13299) (#13301)
+  * When the git ref is unable to be found return broken pr (#13218) (#13303)
+  * Ensure topics added using the API are added to the repository (#13285) (#13302)
+  * Fix avatar autogeneration (#13233) (#13282)
+  * Add migrated pulls to pull request task queue (#13331) (#13334)
```
Comment (6543) [inline CHANGELOG.md:26]:
```suggestion
  * Add migrated pulls to pull request task queue (#13331) (#13334)
  * Issue comment reactions should also check pull type on API (#13349) (#13350)
  * Fix links to repositories in /user/setting/repos (#13360) (#13362)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,7 +4,36 @@ This changelog goes through all the changes that have been made in each release
...
+  * Remove obsolete change of email on profile page (#13341) (#13347)
+  * Fix scrolling to resolved comment anchors (#13343) (#13371)
```
Comment (6543) [inline CHANGELOG.md:30]:
```suggestion
  * Fix scrolling to resolved comment anchors (#13343) (#13371)
  * Storage configuration support `[storage]` (#13314) (#13379)
```

**Review by 6543 — CHANGES_REQUESTED** [review]

**Comment by 6543** [comment]
@techknowlogick this should it bee :)

File: CHANGELOG.md (hunk 6 above)
Comment (6543) [reply CHANGELOG.md:30]: this one is also an backported enhancement ... not sure if we should move it upwards

**Review by 6543 — APPROVED** [review]

**Pull request merged**